src/screens/controllers/UScreenSingController.pas

```diff
@@ -558,10 +559,13 @@ function TScreenSingController.ParseInput(PressedKey: Cardinal; CharCode: UCS4Ch
 
       SDLK_RIGHT:
       begin
-        if (SDL_ModState = KMOD_LCTRL) then // seek 5 seconds forward
-          AudioPlayback.SetPosition(AudioPlayback.Position + 5.0);
-        if (Assigned(fCurrentVideo)) then
-          fCurrentVideo.Position := AudioPlayback.Position + 5.0;
+        if ((SDL_ModState and (KMOD_LCTRL or KMOD_RCTRL)) <> 0) then
```

Comment:
It should not add RCTRL in an unrelated commit. I'm not necessarily against it, but:
- it doesn't have anything to do with fixing videogap (in three years time, you don't expect _this_ to be the commit that introduced a bug by doing so)
- the rest of the file (and possible USDX as a whole) _very consistently_ uses LCTRL and LSHIFT

aka it should be its own thing, outside of this PR.

Reply:
fixed in 220bd3d6f2e797ef8de06a453418b0bbad78e2a3